src/tasks/sep/050-op-betanet-add-game-type/README.md

```diff
@@ -0,0 +1,21 @@
+# 050-op-betanet-add-game-type
+
+Status: [READY TO SIGN]
+
+## Objective
+
+This task adds the dispute game type 0 (Permissionless) to the Dispute Game Factory on OP Betanet for U18.
```

Comment:
Does op-deployer not support configuring permissionless games when the chain is stood up? I thought it did